src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/ModT.txt

```diff
@@ -5,3 +5,33 @@ Table({Value:0},{Value:1})
 
 >> Mod(3, [-1, 0, 1, 2, 3, Blank()])
 Table({Value:0},{Value:Error({Kind:ErrorKind.Div0})},{Value:0},{Value:1},{Value:0},{Value:Error({Kind:ErrorKind.Div0})})
+
+>> Mod([1,2,3,4,5,6,7,8,9,10],3)
```

Comment:
Would add a couple of tests with negative numbers

Reply:
Done.